tests/browser/test/basic_playground_test.js

```diff
@@ -73,8 +79,13 @@ suite('Right Clicking on Blocks', function (done) {
     browser = await testSetup(testFileLocations.playground);
   });
 
+  teardown(async function () {
+    await browser.execute(() => {
+      Blockly.getMainWorkspace().clear();
+    });
+  });
+
   test('Collapse', async function () {
```

Comment:
Could you also go through and update the test titles in this suite to be of the form "this tests that X" where X becomes the title? For example, this test could be come `test('clicking the collapse option collapses the block')`.

Here's an article about why descriptive test names are important! https://testing.googleblog.com/2014/10/testing-on-toilet-writing-descriptive.html